app/src/main/cpp/ExternalVR.cpp

```diff
@@ -179,6 +190,8 @@ struct ExternalVR::State {
   }
 };
 
+ExternalVR::State * ExternalVR::State::sState = nullptr;
+
 ExternalVRPtr
 ExternalVR::Create() {
   return std::make_shared<vrb::ConcreteClass<ExternalVR, ExternalVR::State> >();
```

Comment:
Since the state is no longer instanced with the class this should be:
```
return std::make_shared<ExternalVR>();
```

And `#include "vrb/ConcreteClass.h"` can be removed.